api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirMedicationRequestDaoImpl.java

```diff
@@ -64,4 +108,8 @@ private void handleCodedConcept(Criteria criteria, TokenAndListParam code) {
 		}
 	}
 	
+	private void excludeDiscontinueOrders(Criteria criteria) {
+		// exclude "discontinue" orders, see: https://issues.openmrs.org/browse/FM2-532
+		criteria.add(Restrictions.ne("action", Order.Action.DISCONTINUE));
+	}
```

Comment:
I didn't override the "delete(uuid)" or the "createOrUpdate(DrugOrder)" methods... the delete does a get(uuid) to fetch, so it will now just do nothing if passed the uuid of a discontinue order (which is the same as the current behavior if the uuid didn't match a order at all).

For createOrUpdate, since I didn't touch this, but someone can't create a DISCONTINUE order via FHIR because there's currently no way to represent DISCONTINUE in FHIR.  Someone technically *could* call an update, but this seemed like too much of an edge case to had some convoluted code around... the createOrUpdate would have to check to see if there's an existing drug order with that uuid, and not perform the update if it was type DISCONTINUE.